### DISCUSSION TITLE
Update for Craft 4 compatibility

### DISCUSSION THREAD

**Discussion by aaronmclayton** [trendyminds/algolia#9] [Q&A] [answered]
Is there a timeframe for upgrading this plugin to work with Craft 4? If not, can we help? I utilize this plugin on multiple projects and would like to help if I could.

**Comment by aaronbushnell** [comment, marked as answer]
Hey, @aaronmclayton! Definitely planned as we have some projects we want to get upgraded to 4.0. Stay tuned!

**Reply by aaronbushnell** [reply]
A [new version](https://github.com/trendyminds/algolia/releases/tag/4.0.0) of Algolia is out with support for Craft 4.0!
